k8s/agent-stress-test.yaml

```diff
@@ -0,0 +1,217 @@
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: logdna-agent-ds-priority
+  namespace: logdna-agent
```

Comment:
I think we should run the stress test in a different namespace as to not conflict with an existing agent

Reply:
Right, separate namespace will allow A/B testing and isolation from prod envs.